Makefile

```diff
@@ -227,6 +241,10 @@ push-binary-edge:
 	aws s3 cp opa_windows_$(GOARCH).exe s3://opa-releases/edge/opa_windows_$(GOARCH).exe
 	aws s3 cp opa_linux_$(GOARCH) s3://opa-releases/edge/opa_linux_$(GOARCH)
 
+.PHONY: push-distro-packages-edge
+push-distro-packages-edge:
+	aws s3 cp opa_$(VERSION).deb s3://opa-releases/edge/opa_$(VERSION).deb
+
```

Comment:
Lets remove this for now and we can re-add when we integrate with the CI system.